### ISSUE TITLE
Error when trying to run GraXpert Process or GraXpert Script in PixInsight

### ISSUE THREAD

**Issue by SusanSSB** [Steffenhir/GraXpert#189]
I am getting a "Python error" when I try to run the GraXpert Process or the GraXpert Script in PixInsight. Please refer to the enclosed screenshot of the error. I have been able to run the GraXpert standalone program on my computer without any problems... I hope someone will be able to decipher the error and offer some guidance on how to correct the problem. I would really like to be able to run GraXpert from within PixInsight.

 I run  GraXpert (Release 'Umbrial' 3.0.2) and PixInsight (Core Version 1.8.9-3 Ripley x64 build 1612) on a Gigabyte Technology B650 AORUS Elite AX desktop computer running Windows 11 Pro (Version 23H2) with a AMD Ryzen 9 7900X 12-Core Processor 4.7 GHz, 64.0 GB RAM, an AMD Radeon(TM) Graphics card and a NVIDIA GeForce GTX 1660 SUPER. Let me know if you require any additional information to help diagnose the problem. Thank you.  
 
![Error Ruinning GraXpert](https://github.com/user-attachments/assets/e525e9bf-4741-4803-a505-231e12c0c3e1)

**Comment by schmelly** [comment]
Hi Susan, please try the following:
- if it exists, delete the folder `C:\Users\SusanSB\AppData\Local\GraXpert\GraXpert\bge-ai-models\1.0.1 manually`
- open an image in GraXpert in standalone
- select the background extraction model version in GraXpert's advanced tab that you would like to use (e.g., 1.0.1). If this list is empty, please make sure your system time is correct
- execute the AI-based background extraction from GraXpert standalone. You should get a prompt asking you if you want to download the appropriate ai model. Confirm this.

Afterwards bge from PI should work as expected.

CS, David

**Comment by SusanSSB** [comment]
Thank you for your suggestion... I tried it, but it didn't work. I manually deleted the folder, C:\Users\SusanSB\AppData\Local\GraXpert\GraXpert\bge-ai-models\1.0.1, ran GraXpert in standalone (making sure the Baclround Extration AI-Model 1.0.1 was selected under Advanced settings), but did not get the prompt asking me to download the ai model. Since this did not work, I uninstalled the GraXpert standalone software, rebooted my computer, reinstalled GraXpert and tried running it, but I still was not prompted to download the ai model. How do I go about doing a complete removal of the GraXpert standalone software from my system so I can do a "clean" reinstall? I am thinking that may be the only way to correct the problem. 

Also I should mention that when I ran GraXpert in standalone, it did a great job removing the gradient, but when I tried to try to "Save Background" image, I got an error (see enclosed screenshot below). Note, I had no trouble to "Save Processed" image.
 
![Error Saving Background Gradient GraXpert](https://github.com/user-attachments/assets/d11e6b47-2385-46a4-a648-59da7e6f1959)

**Comment by schmelly** [comment]
Hi Susan, did you select `AI` as the interpolation method? RBF is a method where you manually set samples, similar to DBE in PixInsight.

Please note: when calling GraXpert from PixInsight, only `AI` will be used as interpolation method.

CS, David

**Comment by SusanSSB** [comment]
Hi David:
Okay, I did not select AI as the interpolation method (sigh). Thank you for your help. Though, it is nice to know that the GraXpert Script in PI only uses the AI method. I obviously have more to learn about the standalone version of GraXpert... I will try running Graxpert Background Extraction with the AI and let you know how I make out... Stay tuned.

**Comment by SusanSSB** [comment]
Hi David:

I ran GraXpert Background Extraction on an image using the AI interpolation method, was prompted to download the AI model 1.0.1, as selected under Advanced Settings. It was installed and background extraction completed successfully, WHOO HOO! I also ran Denoising, was prompted to download the selected denoising AI model, it installed and denoising also ran successfully. I then tried both the GraXpert script and the GraXpert process in PI and both work. Although the problems I experienced were largely self-inflicted, thank you very much helping me get things straightened out. :-)
